### PR TITLE
Added documentation for new strava component

### PR DESCRIPTION
**Pull request by stv0g**
**Description:**

Adds documentation for new `strava` component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#22151

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/strava.markdown [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: strava.png
+ha_category:
+  - Sensor
+ha_release: 0.93.0
+ha_iot_class: Cloud Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/strava.markdown]: Please remove this

File: source/_components/strava.markdown [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+description: "Instructions on how to add Strava sensors."
+date: 2019-03-27 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: strava.png
+ha_category:
+  - Sensor
+ha_release: 0.93.0
```
Comment (klaasnicolaas) [inline source/_components/strava.markdown]:
```suggestion
ha_release: 0.93
```

File: source/_components/strava.markdown [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+layout: page
+title: "Strava"
+description: "Instructions on how to add Strava sensors."
+date: 2019-03-27 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: strava.png
+ha_category:
```
Comment (klaasnicolaas) [inline source/_components/strava.markdown:11]: Add also: `Health`

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/strava.markdown (hunk 2 above)
Comment (stv0g) [reply source/_components/strava.markdown]: Done

File: source/_components/strava.markdown (hunk 1 above)
Comment (stv0g) [reply source/_components/strava.markdown]: Done

**Comment by stv0g** [comment]
@klaasnicolaas Hi, I addressed the requested changes :) Can we merge this PR? The component is also ready to be merged in the HA repo.

**Comment by klaasnicolaas** [comment]
The parent pr must be merged first, only then we merge this PR 😉

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed

**Pull request closed without merging**